.github/workflows/release.yml

```diff
@@ -50,6 +50,15 @@ jobs:
           - build: linux-arm
             os: ubuntu-22.04
             target: aarch64-unknown-linux-gnu
+          - build: linux-arm
```

Comment:
@patrickelectric I believe each build name has to be unique included in the matrix build list. I suggest renaming the Linux build jobs to include the architecture & gnu/musl information.